From 920443d3f7afb26f735f4002f3beed8766530d9b Mon Sep 17 00:00:00 2001
From: vixus0 <vixus0@gmail.com>
Date: Tue, 7 Jun 2016 22:06:02 +0100
Subject: [PATCH] Update libsigc++ trackable header import

The `object.h` header was removed in libsigc++ 2.5.2 and `sigc::trackable` is now provided
in `trackable.h`.

Original fix for Arch provided by AUR user sahquievaedoajie.
---
 src/control_osc.hpp         | 2 +-
 src/gui/app_frame.hpp       | 2 +-
 src/gui/config_panel.hpp    | 2 +-
 src/gui/keys_panel.hpp      | 2 +-
 src/gui/latency_panel.hpp   | 2 +-
 src/gui/main_panel.hpp      | 2 +-
 src/gui/midi_bind_panel.hpp | 2 +-
 src/gui/prefs_dialog.hpp    | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/control_osc.hpp b/src/control_osc.hpp
index 11b4e78..d83564f 100644
--- a/src/control_osc.hpp
+++ b/src/control_osc.hpp
@@ -27,7 +27,7 @@
 #include <list>
 #include <utility>
 
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 
 #include "event.hpp"
 #include "event_nonrt.hpp"
diff --git a/src/gui/app_frame.hpp b/src/gui/app_frame.hpp
index e24ff92..36cc18f 100644
--- a/src/gui/app_frame.hpp
+++ b/src/gui/app_frame.hpp
@@ -26,7 +26,7 @@
 #include <string>
 #include <vector>
 
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 #include <sigc++/signal.h>
 #include <sigc++/connection.h>
 
diff --git a/src/gui/config_panel.hpp b/src/gui/config_panel.hpp
index 2b413e9..46c849f 100644
--- a/src/gui/config_panel.hpp
+++ b/src/gui/config_panel.hpp
@@ -26,7 +26,7 @@
 
 #include <string>
 #include <vector>
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 
 class wxListCtrl;
 class wxSpinCtrl;
diff --git a/src/gui/keys_panel.hpp b/src/gui/keys_panel.hpp
index b2c5955..c028c8f 100644
--- a/src/gui/keys_panel.hpp
+++ b/src/gui/keys_panel.hpp
@@ -26,7 +26,7 @@
 
 #include <string>
 #include <vector>
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 
 class wxListCtrl;
 
diff --git a/src/gui/latency_panel.hpp b/src/gui/latency_panel.hpp
index 07dc30f..847da2b 100644
--- a/src/gui/latency_panel.hpp
+++ b/src/gui/latency_panel.hpp
@@ -26,7 +26,7 @@
 
 #include <string>
 #include <vector>
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 
 class wxListCtrl;
 
diff --git a/src/gui/main_panel.hpp b/src/gui/main_panel.hpp
index 0b4ac7d..64cba4c 100644
--- a/src/gui/main_panel.hpp
+++ b/src/gui/main_panel.hpp
@@ -26,7 +26,7 @@
 #include <string>
 #include <vector>
 
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 #include <sigc++/signal.h>
 #include <sigc++/connection.h>
 
diff --git a/src/gui/midi_bind_panel.hpp b/src/gui/midi_bind_panel.hpp
index e919cf2..0d05003 100644
--- a/src/gui/midi_bind_panel.hpp
+++ b/src/gui/midi_bind_panel.hpp
@@ -26,7 +26,7 @@
 
 #include <string>
 #include <vector>
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 #include <list>
 
 #include <midi_bind.hpp>
diff --git a/src/gui/prefs_dialog.hpp b/src/gui/prefs_dialog.hpp
index ea59db1..29fee16 100644
--- a/src/gui/prefs_dialog.hpp
+++ b/src/gui/prefs_dialog.hpp
@@ -26,7 +26,7 @@
 
 #include <string>
 #include <vector>
-#include <sigc++/object.h>
+#include <sigc++/trackable.h>
 
 class wxListCtrl;
 class wxSpinCtrl;
